lms/util/_lti_launch.py

```diff
@@ -23,28 +20,11 @@ def some_view(request):
     @wraps(view_function)
     def wrapper(request):
         """Handle the lms validation."""
-        try:
-            consumer_key = request.params["oauth_consumer_key"]
-        except KeyError:
-            raise MissingLTILaunchParamError("oauth_consumer_key")
-
-        shared_secret = request.find_service(name="ai_getter").shared_secret(
-            consumer_key
-        )
-
-        consumers = {}
-
-        consumers[consumer_key] = {"secret": shared_secret}
-        # TODO rescue from an invalid lms launch
-        pylti.common.verify_request_common(
-            consumers,
-            request.url,
-            request.method,
-            dict(request.headers),
-            dict(request.params),
-        )
+        request.find_service(name="lti").verify_launch_request()
```

Comment:
Catching exceptions and showing an error page belongs here. But it doesn't have it on master and I'm not fixing it in this PR. This entire view decorator is being factored out ASAP anyway